linera-storage/src/lib.rs

```diff
@@ -45,18 +46,36 @@ use linera_views::{
     value_splitting::DatabaseConsistencyError,
     views::{CryptoHashView, RootView, View, ViewError},
 };
-use metrics::increment_counter;
+use prometheus::{register_int_counter_vec, IntCounterVec};
 use serde::{Deserialize, Serialize};
 use std::{fmt::Debug, sync::Arc};
 
-/// The metric counting how often a value is read from storage.
-pub const READ_VALUE_COUNTER: &str = "read_value";
-/// The metric counting how often a value is written to storage.
-pub const WRITE_VALUE_COUNTER: &str = "write_value";
-/// The metric counting how often a certificate is read from storage.
-pub const READ_CERTIFICATE_COUNTER: &str = "read_certificate";
-/// The metric counting how often a certificate is written to storage.
-pub const WRITE_CERTIFICATE_COUNTER: &str = "write_certificate";
+lazy_static! {
+    /// The metric counting how often a value is read from storage.
+    pub static ref READ_VALUE_COUNTER: IntCounterVec = register_int_counter_vec!(
```

Comment:
Gonna try to do these using OnceCell instead

Reply:
Yes please

Reply:
There's also [`OnceLock` in the standard library](https://doc.rust-lang.org/nightly/std/sync/struct.OnceLock.html) now.